data/transactions/logic/eval.go

```diff
@@ -3767,13 +3744,13 @@ func opEcdsaPkDecompress(cx *EvalContext) error {
 	cx.stack[last].Uint = 0
 	cx.stack[last].Bytes, err = leadingZeros(32, x)
 	if err != nil {
-		return fmt.Errorf("x component zeroing failed: %s", err.Error())
+		return fmt.Errorf("x component zeroing failed: %w", err)
```

Comment:
there is no need to wrap since leadingZeros returns untyped error created by `fmt.Errorf`